src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsFloorAdhesionOldObject.java

```diff
@@ -0,0 +1,99 @@
+package com.wikia.webdriver.pageobjectsfactory.pageobject.adsbase;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+import com.wikia.webdriver.common.logging.PageObjectLogging;
+
+public class AdsFloorAdhesionOldObject extends AdsBaseObject {
+
+  private static final String FLOOR_ADHESION_CSS = "#ext-wikia-adEngine-template-floor";
+  private static final String FLOOR_ADHESION_AD_FRAME_CSS =
+      "#ext-wikia-adEngine-template-floor .ad iframe";
+  private static final String FLOOR_ADHESION_IMAGE_IN_FRAME_CSS = "img";
+  private static final String FLOOR_ADHESION_CLOSE_CSS = "#ext-wikia-adEngine-template-floor .close";
+  private static final String WIKIA_BAR_CSS = "#WikiaBar";
+
+  public AdsFloorAdhesionOldObject(WebDriver driver, String testedPage) {
+    super(driver);
+    getUrl(testedPage, true);
+  }
+
+  public void verifyFloorAdhesionPresent(String expectedSlotName, String expectedLineItemId,
+                                         String expectedCreativeId) {
```

Comment:
Is this how our convention describes a method signature? If yes, then fine but it looks awkward for me. However, it could be something wrong with me :japanese_ogre: I'd go with one of those two:
```
public void verifyFloorAdhesionPresent(
   String expectedSlotName, 
   String expectedLineItemId,
   String expectedCreativeId) 
{
```
or
```
public void verifyFloorAdhesionPresent(String expectedSlotName, String expectedLineItemId, String expectedCreativeId) {
```
but this one gets too long IMHO.

What do you think?